metadata.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: postgresql-k8s
+display-name: PostgreSQL K8s
+description: |
+  Charm to operate the PostgreSQL database on Kubernetes clusters
+summary: |
+  Charm to operate the PostgreSQL database on Kubernetes clusters
+
+containers:
+  postgresql:
+    resource: postgresql-image
+    mounts:
+      - storage: pgdata
+        location: /var/lib/postgresql/data/pgdata
+
+resources:
+  postgresql-image:
+    type: oci-image
+    description: OCI image for PostgreSQL (ubuntu/postgres)
+    upstream-source: ubuntu/postgres:12-20.04_beta
+
+storage:
+  pgdata:
+    type: filesystem
+    location: /var/lib/postgresql/data/pgdata
```

Comment:
Might want to add `peer` info here
```
peers:
  postgresql:
    interface: postgresql
```

Reply:
Agreed - might be worth naming it `postgresql-replicas` or similar, though

Reply:
Thanks Mia and Jon. I created the peer relation and submitted in the last commit.